### DISCUSSION TITLE
Can't create hub files

### DISCUSSION THREAD

**Discussion by StrgCStrgD** [ScalefreeCOM/turbovault4dbt#10] [Q&A]
Hi everyone,

i was able to create the sources.yml file and multiple stage files. I've also used the stage files to create the stages in Snowflake.
When i try to create the hubs the scripts runs through but no files get created.
I already tried a few things but i cant solve it, because there is no error message. Just nothing happens.

**Comment by universe-designer** [comment]
I had a similar problem. Have you set the columns as followed?
record_tracking_satellite=0
is_primary_source=1
group_name=group1. 

The models are created. However I run into another [problem](https://github.com/ScalefreeCOM/turbovault4dbt/issues/12)
